superbench/benchmarks/micro_benchmarks/cuda_memory_bw_performance.py

```diff
@@ -108,23 +71,17 @@ def _process_raw_result(self, cmd_idx, raw_output):
         self._result.add_raw_data('raw_output_' + self._args.mem_type[cmd_idx], raw_output)
```

Comment:
It's too weird to access variable that is not declared. Could we put it into __init__?